test/Syntax/Parser/unterminated_multiline_regex.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %swift-syntax-parser-test -dump-diags %s | %FileCheck %s
+// CHECK: 8:1 Error: unterminated regex literal
+// CHECK: 1 error(s) 0 warnings(s) 0 note(s)
+
+// IMPORTANT: This file must not contain a trailing newline
+#/
+unterminatedLiteral
```

Comment:
FYI GitHub doesn't appear to show the "no newline at end of file" marker for this case

Given the fiddly nature of these tests, I kind of wonder whether we should at some point introduce a testing script that strips trailing newlines, rather than having the file be like that

Reply:
Oh, that’s because this file is supposed to have a newline (because it’s using a multi-line regex literal). I just forgot to remove the comment.